src/overlays/actors/ovl_Obj_Tokeidai/z_obj_tokeidai.c

```diff
@@ -36,7 +36,7 @@
 
 #define THIS ((ObjTokeidai*)thisx)
 
-#define GET_CURRENT_CLOCK_HOUR(this) ((s32)((this)->clockTime * (24.0f / 0x10000)))
+#define GET_CURRENT_CLOCK_HOUR(this) ((s32)TIME_TO_HOURS_F((this)->clockTime))
 #define GET_CURRENT_CLOCK_MINUTE(this) ((s32)((this)->clockTime * (360 * 2.0f / 0x10000)) % 30)
 #define GET_CLOCK_FACE_ROTATION(currentClockHour) ((s16)(currentClockHour * (0x10000 / 24.0f)))
 #define GET_MINUTE_RING_OR_EXTERIOR_GEAR_ROTATION(currentClockMinute) \
```

Comment:
Is there any scope for improving these based on the new time macros (if we even still need all of them)?

Reply:
`GET_CURRENT_CLOCK_HOUR` could just be replaced entirely with `TIME_TO_HOURS_F`.
`GET_CURRENT_CLOCK_MINUTE` differs from `TIME_TO_MINUTES_F` by a factor of 2, and that makes the macro unusable here because `TIME_TO_MINUTES_F(this->clockTime * 2.0f) messes up the grouping of the constants